cmd/jaeger/main.go

```diff
@@ -19,7 +20,7 @@ func main() {
 	command := internal.Command()
 	command.AddCommand(version.Command())
 	command.AddCommand(docs.Command(v))
-
+	command.AddCommand(printconfig.Command(v))
```

Comment:
I don't think we need it here, jaeger-v2 is not configured via CLI flags. In fact, its print-config would be implemented differently (there's no solution for it yet)